setup.py

```diff
@@ -24,5 +25,8 @@
     author_email='oscm@edx.org',
     license='Apache',
     packages=find_packages(exclude=['*.tests']),
-    install_requires=['slumber >= 0.7.1, < 1.0', 'PyJWT >= 1.1.0, < 2.0'],
+    install_requires=[
+        'slumber>=0.7.1',
+        'PyJWT>= 1.1.0'
```

Comment:
We've been having discussions recently, via email and in-person, about requirements. The general consensus has been to make requirements less-restrictive to avoid issues with one package being stuck on a lower version. Happy to discuss further.